test/unit/components/chart/daily.test.js

```diff
@@ -0,0 +1,105 @@
+/* global chai */
+
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var _ = require('lodash');
+var expect = chai.expect;
+
+var Daily = require('../../../../app/components/chart/daily');
+
+require('tideline/css/tideline.less');
+require('../../../../app/core/less/fonts.less');
+require('../../../../app/style.less');
+window.localStorage.debug = true;
+
+describe('Daily', function () {
+  describe('render', function() {
```

Comment:
Yes, that would help greatly! As soon as we can mock, the testing world is our oyster!